storio-content-resolver-annotations/src/main/java/com/pushtorefresh/storio/contentresolver/annotations/StorIOContentResolverType.java

```diff
@@ -0,0 +1,22 @@
+package com.pushtorefresh.storio.contentresolver.annotations;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * Annotation for marking class as StorIOContentResolverType to generate object mapping code
+ */
+@Target(TYPE)
+@Retention(RUNTIME) // we allow users to write reflection based code to work with annotation
+public @interface StorIOContentResolverType {
```

Comment:
Ook, but should we provide the default `uri` for all operations? if user doesn't want to set it separately?
Also, it can allow to make 1.7 without braking backward campatibility